libraries/singleton-devices/CMakeLists.txt

```diff
@@ -8,4 +8,9 @@ target_link_libraries(Handler
   PRIVATE
     ${YARP_LIBRARIES}
     YARP::YARP_os
-    YARP::YARP_init)
\ No newline at end of file
+    YARP::YARP_init)
+
+install(TARGETS Handler
+        RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
```

Comment:
Two suggestions:
* We are using the `CMAKE_INSTALL_BINDIR` variable, that is defined by the `GNUInstallDirs` module (see https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html), but we are not including anywhere the module (i.e. via `include(GNUInstallDirs)`). Probably everything works as that module is already included either by YARP or Gazebo config cmake, but to be sure it is better to include it (see for example https://github.com/robotology/how-to-export-cpp-library/blob/afb21efb655e7b1cc11636c3d42aad9e02fb626f/CMakeLists.txt#L32)
* As we are using CMake 3.16 as minimum cmake version, the `DESTINATION` variables have a default value that is the one that we are setting (see https://cmake.org/cmake/help/v3.16/command/install.html#installing-targets). So probably instead of listing all the `*** DESTINATION` parameters, we can just add `install(TARGET Handler)`.